addons/music/fnc_getMusicData.sqf

```diff
@@ -0,0 +1,32 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getMusicData
+Description:
```

Comment:
Header is missing some line-breaks see [this](https://github.com/fishykins/CBA_A3/blob/master/addons/common/fnc_addBackpackCargo.sqf) as example